x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/enrich/AbstractLookupService.java

```diff
@@ -459,6 +460,7 @@ abstract static class Request {
     abstract static class TransportRequest extends org.elasticsearch.transport.TransportRequest implements IndicesRequest {
         final String sessionId;
         final ShardId shardId;
+        @Nullable
```

Comment:
Could you add a javadoc to this one saying it's null for versions before <WHATEVER VERSION IT IS>?

Reply:
Done

Reply:
Ah! `&lt;`. I thought that might be a problem.....